pkg/util/log.go

```diff
@@ -44,8 +44,8 @@ func InitLogger(cfg *server.Config) {
 		panic(err)
 	}
 
-	Logger = l
-	cfg.Log = logging.GoKit(l)
+	Logger = log.With(l, "caller", log.DefaultCaller)
+	cfg.Log = logging.GoKit(log.With(l, "caller", log.Caller(5)))
```

Comment:
Do we really want both `DefaultCaller` and `log.Caller` ?

Reply:
They have different stack depth. `DefaultCaller` is equivalent to `log.Caller(3)`.

Reply:
OK, but do we want two of them on every log line?

Reply:
oh, these are going to different places?

Reply:
Yes. There won't be two caller information in a single log message.